mwa_trigger/parse_xml.py

```diff
@@ -144,13 +147,13 @@ def parse(self):
 
         # See if the trigger has a source name
         if self.telescope == "SWIFT" and self.event_type == "BAT_GRB_Pos":
-            self.source_name = v.Why.Inference.Name
+            self.source_name = str(v.Why.Inference.Name)
         elif self.telescope == "MAXI":
             # MAXI uses a Source_Name parameter
             src = v.find(".//Param[@name='Source_Name']")
             if src is not None:
                 # MAXI sometimes puts spaces at the start of the string!
-                self.source_name = src.attrib['value'].strip()
+                self.source_name = str(src.attrib['value'].strip())
```

Comment:
lots of `str` casting going on which may be needed. However, this particular casting isn't required since the inner `.strip()` will not work if the variable insn't already a string. Maybe you mean to put the `.strip()` outside the `str()`?

Reply:
I used the str casting because it seemed to use some lxml.objectify str type which assert_equal didn't like. I can put the .strip() outside